app/models.py

```diff
@@ -983,9 +983,14 @@ def _as_utils_template(self):
         if self.template_type == LETTER_TYPE:
             return LetterPrintTemplate(
                 self.__dict__,
-                contact_block=self.service.get_default_letter_contact(),
+                contact_block=self.get_reply_to_text(),
```

Comment:
No – this is a change to the `_as_utils_template_method` to make it possible to use it for checking personalisation. Without this change it would always be looking for placeholders in the service’s default contact block, not the contact block for that specific template.